.github/workflows/docusaurus.yaml

```diff
@@ -0,0 +1,36 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+
+      - name: Install dependencies
+        run: cd docusaurus/ && yarn install --frozen-lockfile
+      - name: Build website
+        run: cd docusaurus/ && yarn build
+
+      # Popular action to deploy to GitHub Pages:
+      # Docs: https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@bisht-richa Can we change this to:

`deploy_key: ${{ secrets.GH_PAGES_DEPLOY_KEY }}`

We can then use a deployment key rather that a token.

Reply:
Thank you.